apps/worker/src/app/workflow/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -136,49 +131,33 @@ export class SendMessageEmail extends SendMessageBase {
       return;
     }
 
-    let actor: SubscriberEntity | null = null;
-    if (command.job.actorId) {
-      actor = await this.getSubscriberBySubscriberId({
-        subscriberId: command.job.actorId,
-        _environmentId: command.environmentId,
-      });
-    }
-
-    const [tenant, overrideLayoutId] = await Promise.all([
-      this.handleTenantExecution(command.job),
+    const [template, overrideLayoutId] = await Promise.all([
+      this.processVariants(command),
       this.getOverrideLayoutId(command),
       this.sendSelectedIntegrationExecution(command.job, integration),
     ]);
 
+    if (template) {
+      step.template = template;
+    }
+
     const overrides: Record<string, any> = Object.assign(
       {},
       command.overrides.email || {},
       command.overrides[integration?.providerId] || {}
     );
 
     let html;
-    let subject = '';
+    let subject = step?.template?.subject || '';
     let content;
-    let senderName = overrides?.senderName || emailChannel.template.senderName;
```

Comment:
[The sender name allowing variables](https://github.com/novuhq/novu/pull/4184) has somehow been removed in this file. Probably a mix up when merging conflicts

Reply:
@ainouzgali the variable was removed, but instead `overrides?.senderName || emailChannel.template.senderName` is passed to the `sendMessage` function in the line 288